lib/host-writer.js

```diff
@@ -41,9 +41,15 @@ module.exports = class HostWriter {
   untilExit(hosts) {
     return this.point(hosts).then(() => {
       const resetAndExit = () => {
-        return this.point(mapObject(hosts, () => null)).then(() => {
-          process.exit(1); // eslint-disable-line no-process-exit
-        });
+        return this.point(mapObject(hosts, () => null))
+          .catch(err => {
+            console.warn(
+              'Could not automatically reset host file entry. Make sure the file is writable to enable this feature.',
+              err.stack);
```

Comment:
is stack trace useful in this case? For those that don't plan to make it writable, seems it might be overly verbose.

Reply:
Good point.